x/poolx/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,40 @@
+package keeper_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/cosmos/cosmos-sdk/x/poolx/keeper"
+	"github.com/cosmos/cosmos-sdk/x/poolx/types"
+	"github.com/stretchr/testify/require"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+func TestKeeperParams(t *testing.T) {
+	app := simapp.Setup(false)
+	ctx := app.BaseApp.NewContext(false, tmproto.Header{})
+
+	var ( // TODO(levi) figure how how to initialize these -- footnote: interesting that the test passes and doesn't care
+		cdc        codec.Codec
+		storeKey   sdk.StoreKey
+		memKey     sdk.StoreKey
+		paramSpace paramtypes.Subspace
+	)
```

Comment:
These values are required to call `NewKeeper` -- I wasn't sure how to initialize them & it seems like the zero values are good enough for the current test. Any pointers to a test that knows how to init these would be helpful though.